macros/LearnSidebar.ejs

```diff
@@ -693,6 +707,7 @@ var text = mdn.localString({
     <ol>
       <li><a href="<%=baseURL%>HTML/Howto"><%=text['HTML_questions']%></a></li>
       <li><a href="<%=baseURL%>CSS/Howto"><%=text['CSS_questions']%></a></li>
+      <li><a href="<%=baseURL%>JavaScript/Howto"><%=text['JavaScript_questions']%></a></li>
```

Comment:
This is "How_to", but you could also move the page and be consistent with the HTML and CSS ones https://developer.mozilla.org/en-US/docs/Learn/JavaScript/How_to